base_classes/nyaml/NXdata.yaml

```diff
@@ -193,6 +193,18 @@ NXdata(NXobject):
       
       If there are no axes at all (such as with a stack of images),
       the axes attribute can be omitted.
+  \@reference:
```

Comment:
Why not an attribute of DATA (to be consistent with AXISNAME/@reference)

Reply:
This exists as well. I think this attribute is meant to link entire NXdata groups together, not just DATA or AXISNAME fields.